### PR TITLE
Update Netlify Logo

### PR DESCRIPTION
**Pull request by lorenzolewis**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

- Update Netlify Logo based on their rebrand: https://www.netlify.com/blog/netlify-new-logo/ https://www.netlify.com/press/#logos
- Note that Netlify CMS has been spun-off/rebranded so not included in this PR: https://www.netlify.com/blog/netlify-cms-to-become-decap-cms/

### CODE REVIEW

File: public/logos/netlify.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 113"><g clip-path="url(#a)"><path d="M34.593 94.05h-1.209l-6.033-6.032v-1.209l9.223-9.222h6.39l.852.852v6.39l-9.223 9.222ZM27.351 25.816v-1.209l6.033-6.033h1.209l9.223 9.223v6.39l-.852.852h-6.39l-9.223-9.223Z" fill="#32E6E2"/><path d="M80.46 74.605h-8.778l-.733-.733V53.326c0-3.656-1.436-6.489-5.844-6.588-2.269-.06-4.865 0-7.638.11l-.416.425v26.589l-.733.733H47.54l-.733-.733V38.764l.733-.733h19.753c7.677 0 13.898 6.22 13.898 13.898v21.943l-.733.733Z" fill="#fff"/><path d="M35.841 61.45H.733L0 60.715v-8.797l.733-.733h35.108l.733.733v8.797l-.733.733ZM127.277 61.45H92.169l-.733-.734v-8.797l.733-.733h35.108l.733.733v8.797l-.733.733ZM58.943 27.064V.734L59.676 0h8.797l.733.733v26.331l-.733.733h-8.797l-.733-.733ZM58.943 111.902v-26.33l.733-.734h8.797l.733.733v26.331l-.733.733h-8.797l-.733-.733Z" fill="#32E6E2"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h128v112.635H0z"/></clipPath></defs></svg>
```
Comment (delucis) [inline public/logos/netlify.svg:1]: Suggesting an even more compressed logo!

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 128 113"><g clip-path="url(#a)"><path fill="#32E6E2" d="M34.6 94h-1.2l-6-6v-1.2l9.2-9.2H43l.8.8v6.4l-9.2 9.3Zm-7.2-68.2v-1.2l6-6h1.2l9.2 9.2v6.4l-.8.8h-6.4l-9.2-9.2Z"/><path fill="#fff" d="M80.5 74.6h-8.8l-.8-.7V53.3c0-3.6-1.4-6.5-5.8-6.6l-7.6.1-.4.5v26.6l-.8.7h-8.8l-.7-.7V38.8l.7-.8h19.8a14 14 0 0 1 13.9 14v21.9l-.7.7Z"/><path fill="#32E6E2" d="M35.8 61.5H.8l-.8-.8V52l.7-.7h35.1l.8.7v8.8l-.8.7Zm91.5 0H92.2l-.8-.8V52l.8-.7h35l.8.7v8.8l-.7.7ZM58.9 27V.7l.8-.7h8.8l.7.7v26.4l-.7.7h-8.8l-.8-.7Zm0 84.8V85.6l.8-.8h8.8l.7.8v26.3l-.7.7h-8.8l-.8-.7Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h128v112.6H0z"/></clipPath></defs></svg>
```

File: src/components/LeftSidebar/Sponsors.astro [hunk 2]
```diff
@@ -5,22 +5,14 @@ import UIString from '../UIString.astro';
...
+				></path>
+			</g><defs><clipPath id="a"><path d="M0 0h256v104.311H0z"></path></clipPath></defs>
 		</svg>
```
Comment (delucis) [inline src/components/LeftSidebar/Sponsors.astro:16]: Same here, plus important to add `aria-hidden="true"` here to hide the SVG — we provide the accessible text via the `aria-label` attribute on the link. I also felt the new logo felt smaller at 90px wide compared to the Storyblok one, so bumped that up to 105px.

```suggestion
		<svg xmlns="http://www.w3.org/2000/svg" width="105" viewBox="0 0 256 105" aria-hidden="true">
			<path fill="#32E6E2" d="M58.5 103.8V77.4l.5-.5h6.6l.6.5v26.4l-.6.5H59l-.5-.5Zm0-76.9V.5L59 0h6.6l.6.5V27l-.6.5H59l-.5-.5ZM35.8 85.2h-1l-4.4-4.5v-.9l8.5-8.5h4.7l.7.6v4.8l-8.5 8.5Zm-5.4-60.5v-.9l4.5-4.5h.9l8.5 8.5v4.8l-.7.6H39l-8.5-8.5ZM.5 48.3H38l.5.6v6.6l-.5.5H.5l-.5-.5v-6.6l.5-.6Zm255 0 .5.6v6.6l-.6.5h-37.8l-.6-.5 2.8-6.6.5-.6h35.1Z"></path>
			<path d="M74.7 65.9H68l-.6-.6V50c0-2.7-1-4.9-4.4-5-1.7 0-3.6 0-5.7.2l-.3.3v20l-.6.5H50l-.6-.6V39l.6-.6h14.8c5.7 0 10.4 4.7 10.4 10.5v16.4l-.5.6Zm31.9-11.6-.6.6H89l-.6.5c0 1.1 1.1 4.4 5.5 4.4 1.7 0 3.3-.5 3.9-1.6l.5-.6h6.6l.6.6c-.6 3.3-3.3 8.2-11.6 8.2-9.3 0-13.7-6.6-13.7-14.3S84.6 38 93.4 38s13.2 6.6 13.2 14.2v2.2ZM98.3 49c0-.6-.5-4.4-5-4.4s-4.9 3.8-4.9 4.4l.6.5h8.8l.5-.5Zm23.7 8.7c0 1.1.5 1.7 1.6 1.7h5l.5.5v5.5l-.6.6h-4.9c-5 0-9.3-2.2-9.3-8.3v-12l-.6-.6H110l-.6-.5V39l.6-.6h3.8l.6-.5v-5l.5-.5h6.6l.5.5v5l.6.5h6l.6.6v5.5l-.6.5h-6l-.6.6v12Zm20.3 8.3h-6.6l-.6-.6V28l.6-.6h6.6l.5.6v37.3l-.5.6ZM157 34h-6.6l-.5-.5V28l.5-.6h6.6l.6.6v5.5l-.6.5Zm0 31.9h-6.6l-.5-.6V39l.5-.6h6.6l.6.6v26.3l-.6.6Zm25.8-38v5.6l-.5.5h-5c-1 0-1.6.6-1.6 1.7v2.2l.5.5h5.5l.6.6v5.5l-.6.5h-5.5l-.5.6v19.7l-.6.6h-6.5l-.6-.6V45.5l-.5-.5h-3.9l-.5-.5V39l.5-.6h3.9l.5-.5v-2.2c0-6 4.4-8.3 9.4-8.3h4.9l.5.6Zm20.3 38.5c-2.2 5.5-4.3 8.8-12 8.8h-2.8l-.5-.5v-5.5l.5-.6h2.8c2.7 0 3.3-.5 3.8-2.2V66l-8.8-21.4V39l.6-.6h5l.5.6 6.5 18.7h.6L206 39l.5-.6h5l.5.6v5.5l-8.8 22Z"></path>
		</svg>
```

**Review by delucis — COMMENTED** [review]
Thanks @lorenzolewis! Left a couple of small tweak suggestions — super helpful to have these updated.

File: src/components/LeftSidebar/Sponsors.astro (hunk 2 above)
Comment (lorenzolewis) [reply src/components/LeftSidebar/Sponsors.astro:16]: Cool, I thought it was small compared to the old one and next to Storyblok but wasn't sure if you all would want it bumped. Thanks!

**Review by delucis — APPROVED** [review]
Perfect! Thank you again @lorenzolewis — and happy official welcome to Team Docs 🥳

**Pull request merged**